standalone_SCINGE.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
We should add documentation to the main readme showing how to run this script the the example data.  This should explain the multiple ways to run SINGE, either with the bash script and compiled MATLAB or with a MATLAB file like `SINGE_Example`.

Also, let's use `SINGE` in the script name.